src/bilder/images/edxapp/custom_install.pkr.hcl

```diff
@@ -93,6 +93,31 @@ build {
     ]
   }
 
+  dynamic "provisioner" {
+    for_each = ((var.installation_target == "mitx" || var.installation_target == "mitx-staging") && var.node_type == "web") ? ["this"] : []
+    labels   = ["shell"]
+    content {
+      environment_vars = [
+        "EDX_ANSIBLE_BRANCH=${var.edx_platform_version}",
+      ]
+      inline = [
+        "openssl req -new -newkey rsa:2048 -days 365 -nodes -x509 -keyout /tmp/edxapp.key -out /tmp/edxapp.cert -subj '/C=US/ST=MA/L=Cambridge/O=MIT Open Learning/OU=Engineering/CN=edxapp.example.com'",
+        "cd /tmp && git clone https://github.com/edx/configuration --depth 1 --branch $EDX_ANSIBLE_BRANCH",
+        "cd /tmp/configuration && python3 -m venv .venv && .venv/bin/pip install wheel && .venv/bin/pip install -r requirements.txt",
+      ]
+    }
+  }
+  dynamic "provisioner" {
+    for_each = ((var.installation_target == "mitx" || var.installation_target == "mitx-staging") && var.node_type == "web") ? ["this"] : []
+    labels   = ["ansible-local"]
+
+    content {
```

Comment:
Couldn't this part of the same `provisioner` above since it's the same conditional?

Reply:
I think it needs to be separate because the `labels` attribute ends up getting mapped to the provisioner type for the block.